llvm/utils/git/github-automation.py

```diff
@@ -113,6 +122,11 @@ def run(self) -> bool:
             print(f"couldn't find team named {self.team_name}")
             return False
 
+        # GitHub limits comments to 65,536 characters, let's limit the diff
+        # and the file list to 20kB each.
+        STAT_LIMIT = 20 * 1024
+        DIFF_LIMIT = 20 * 1024
+
```

Comment:
This isn't in the PR description, can you add it there? (with some context for the "why" of the change)